cwl/src/main/scala/cwl/preprocessor/CwlPreProcessor.scala

```diff
@@ -100,6 +108,8 @@ object CwlPreProcessor {
   */
 class CwlPreProcessor(saladFunction: BFile => Parse[String] = saladCwlFile) {
 
+  implicit val composedApplicative = Applicative[IO] compose Applicative[ErrorOr]
```

Comment:
Please type annotate public implicit vals

Reply:
Oh wait, is this contained to the `class`? Either way, I think an annotation would be nice :)